packages/apps/dashboard/server/src/modules/details/dto/details-pagination.dto.ts

```diff
@@ -9,9 +9,47 @@ import {
   IsIn,
 } from 'class-validator';
 import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
-import { ChainId } from '@human-protocol/sdk';
+import { ChainId, OrderDirection } from '@human-protocol/sdk';
 
 import { IsRoleValid } from './validation/role-validation';
+import { LeadersOrderBy } from 'src/common/enums/leader';
+
+export class LeadersPaginationDto {
+  @ApiProperty({ enum: ChainId })
+  @IsEnum(ChainId)
+  @IsIn(Object.values(ChainId).filter((id) => id !== ChainId.ALL))
```

Comment:
Please remove the `All Networks` option on UI in scope of this PR and as a default one make the first in the list of available options, otherwise this PR will break leaders table